ci-operator/config/openshift/openshift-azure/master.json

```diff
@@ -1,8 +1,10 @@
 {
   "tag_specification": {
     "cluster": "https://api.ci.openshift.org",
-    "namespace": "azure",
-    "name": "plugin-base"
+    "namespace": "openshift",
+    "name": "origin",
```

Comment:
Why this change?

Reply:
This is thing we didn't do, but we suppose to. This tags all particular release images into our ci-namespace for us to use. If we need ansible,origin, or any other image from another team - this is how we get them so we dont need to build it ourselves.